crates/solidity/inputs/language/src/definition.rs

```diff
@@ -4,7 +4,6 @@ codegen_language_macros::compile!(Language(
     name = Solidity,
     documentation_dir = "crates/solidity/inputs/language/docs",
     binding_rules_file = "crates/solidity/inputs/language/bindings/rules.msgb",
```

Comment:
I suggest adding a `minor` changeset for the change to use contexts in bindings, and a `patch` changeset for the #1244 fix.